src/utils.ts

```diff
@@ -62,6 +62,12 @@ export function dayToMillisec(dayAmount: number) {
   return dayAmount * 24 * 60 * 60 * 1000
 }
 
+export function getMonthsBetweenDates(startDate: Date, endDate: Date) {
```

Comment:
Let's add some tests for this fn please :) there might me some interesting edge cases